modules/alibaba-ai-converting-image-to-qcow2.adoc

```diff
@@ -11,19 +11,30 @@ Convert the generated ISO to `QCOW2` format before importing it into {alibaba}.
 .Prerequisites
 
 * You have created a cluster and downloaded the discovery image in the {ai-full}.
-* You have access to a Red Hat Enterprise Linux or Fedora machine that is outside the cluster, such as your desktop machine.
-* You have ensured that the machine has virtualization flags enabled.
+* You have access to a Red Hat Enterprise Linux (RHEL) or Fedora machine that is outside the cluster, such as your desktop machine. 
 
 .Procedure
 
 . Open the command line interface on your RHEL or Fedora machine.
 
+. Verify that the system has virtualization flags enabled by running the following command:
++
+[source,terminal]
+----
+$ grep -e lm -e svm -e vmx /proc/cpuinfo
+----
+
 . Verify that `qemu-img` is installed on the machine by running the following command:
 +
 [source,terminal]
 ----
 $ sudo dnf install -y qemu-img
 ----
++
+[NOTE]
```

Comment:
I'm not sure I fully understand the NOTE callout here. Is this actually an optional step to the previous one? More broadly, I'm confused about this procedures steps which are, to simplify: open terminal, verify x, verify y, and convert. In that order, I don't see where the user is being asked to install anything which is where the NOTE's confusion comes from because it seems out of context with the procedure itself.

Reply:
Hi @JoeAldinger I changed it.